core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -230,6 +258,7 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
     val producer1 = ProducerBuilder().trustStoreProps(sslProperties2).maxRetries(0).build()
     verifyAuthenticationFailure(producer1)
 
+    sslProperties2.setProperty("ssl.keystore.password", "${file:/usr/kerberos:password}")
```

Comment:
as mentioned in the previous comment, we should revert this.

Reply:
reverted it to original version.